Cloud_Providers.md

```diff
@@ -0,0 +1,50 @@
+
+## Microsoft Azure
+
+### Deploying SAS Viya Logging and Monitoring on Azure
+
+In order to deploy SAS Viya logging and monitoring on Azure, you must create a custom Storage Class. 
+Follow the procedures in the [Azure sample](/samples/azure-deployment/README.md) for information on customizing your deployment for Azure.
+
+You must also deploy logging and monitoring using ingress.
+
+### Using Azure's Logging and Monitoring Applications With SAS Viya
+
+If you deploy SAS Viya in an Azure environment, you can monitor and view log information by using either Azure Monitor or the SAS Viya logging and monitoring components. In order to determine which 
+solution is right for your environment, you must consider the monitoring needs of both the cluster administrator and the SAS Viya administrator and compare the capabilities of the monitoring solutions. See [Choose a Monitoring Strategy for an Azure Environment](https://go.documentation.sas.com/?cdcId=sasadmincdc&cdcVersion=v_001LTS&docsetId=calmonitoring&docsetTarget=n0okndfnjiwcxmn1ci29kjroaamp.htm&locale=en#n006zgl0bnp8txn15azzj13z8ww8) in the SAS Viya documentation for complete information. 
+
+## Amazon Elastic Kubernetes Service (EKS)
+
+### Deploying SAS Viya Logging and Monitoring on Amazon EKS
+
+In order to use the SAS Viya monitoring and logging components in an Amazon EKS environment, you must deploy monitoring and logging using ingress. See the [Ingress sample](/samples/ingress/README.md) for deployment information.
+
+No other customization is needed to deploy and run the SAS Viya logging and monitoring components on Amazon EKS.
+
+### Using Amazon EKS Logging and Monitoring Applications With SAS Viya
+
+If you deploy SAS Viya in an Amazon EKS environment, you can view metric information by using either the SAS Viya monitoring components or Amazon's monitoring application, CloudWatch. In order to use CloudWatch to monitor SAS Viya metrics, you must deploy the CloudWatch agent for Prometheus, make configuration changes to convert the Prometheus metrics collected from SAS Viya into performance log events, and map the log events to CloudWatch metrics. Follow the steps in the [Amazon CloudWatch Integration sample](/samples/cloudwatch/README.md) to use CloudWatch to monitor a selected group of SAS Viya metrics. Not all SAS Viya metrics are supported by this sample, because supporting all SAS Viya metrics would require every possible combination of labels from every SAS Viya source and every metric to be mapped to CloudWatch metric types. This sample might be updated in later releases to include additional combinations of dimensions for key metrics.
+
+Amazon CloudWatch can also display log messages collected from Fluent Bit, which is the application used by the SAS Viya logging solution. However, current samples do not provide the configuration needed to integrate SAS Viya log processing with CloudWatch.  
```

Comment:
Amazon Cloud Watch actually uses Fluent Bit for its own log collection component within it CloudWatch Logs offering.  And, some customers may choose to deploy the CloudWatch Logs offering as their only tool for monitoring their Viya deployment.  And that will work just fine.  However, just as was the case with Azure Monitor's log collection/monitoring process, the trade-off is that not all of the components of SAS Viya generate log messages in the same way.  This means the log messages collected by CloudWatch Logs will not appear formatted the same way.  On the other hand, deploying the Viya 4 Monitoring solution (or even just deploying Fluent Bit with our configuration files) will mean customers will see most log messages generated by all of the components of Viya handled and presented consistently, even if the original log message wasn't.  It is almost definitely true that a sample demonstrating this level of integration (i.e. the middle ground of deploying Fluent Bit with our configuration logic outputting to CloudWatch Logs) will be delivered at some point in the future.  But, since I don't think we want to say that, I'd vote for omitting a mention of it rather than highlighting that it doesn't exist.

Reply:
Changed the last sentence to "Although these messages are processed by the SAS Viya logging applications so that Kibana displays them using a consistent format, these messages are displayed in different formats on CloudWatch."